var/spack/repos/builtin/packages/go/package.py

```diff
@@ -123,19 +127,19 @@ def setup_dependent_package(self, module, dependent_spec):
         #  Add a go command/compiler for extensions
         module.go = self.spec['go'].command
 
-    def setup_dependent_environment(self, spack_env, run_env, dependent_spec):
+    def setup_run_environment(self, env):
```

Comment:
I think this should be `setup_dependent_run_environment`. See https://spack.readthedocs.io/en/latest/tutorial_advanced_packaging.html#modifying-a-package-s-build-environment

Reply:
I was getting an error during CI informing me those had been deprecated (https://github.com/spack/spack/pull/11115). I was incorrect in my use of `setup_run_envrionment` but I didn't have time yesterday to correct it.

Reply:
Yep, `setup_environment` is now `setup_build_environment` and `setup_run_environment`, while `setup_dependent_environment` is now `setup_dependent_build_environment` and `setup_dependent_run_environment`.

Reply:
Completely missed that, okay, I'll correct that and re-test. Thanks